src/app/search-page/card/card.directive.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Directive, ElementRef, Input, OnInit,
+} from '@angular/core';
+import { dateDifferenceInDays, dateDifferenceInMonths } from 'src/app/shared/utils/dateUtils';
+
+@Directive({
+  selector: '[appCard]',
+})
+class CardDirective implements OnInit {
+  @Input() publishDate : Date = new Date();
+
+  constructor(private el: ElementRef) {
+
+  }
+
+  ngOnInit() {
+    const currentDate : Date = new Date();
+    const days = dateDifferenceInDays(new Date(this.publishDate), currentDate);
+    const months = dateDifferenceInMonths(new Date(this.publishDate), currentDate);
+    if (months > 0) {
+      this.el.nativeElement.style.borderColor = 'yellow';
```

Comment:
could be extracted into `enum` or `const` for reusability :) (basically having "magic strings" on project is not really a good practise)